setup.cfg

```diff
@@ -63,7 +63,10 @@ include = torchgeo*
 # Optional dataset requirements
 datasets =
     h5py
+    # loading .las point clouds (idtrees)
+    laspy
     opencv-python
+    pandas
```

Comment:
What minimum versions of these dependencies are required?

Reply:
pip install pandas<1.0  on Ubuntu 20.04 Python 3.9 gives me CPython errors. Looks like laspy only [recently](https://github.com/laspy/laspy/releases/tag/2.0.0) got Python 3.6+ support in v2.0.0.